schutzbot/mockbuild.sh

```diff
@@ -27,13 +27,7 @@ fi
 
 # Install requirements for building RPMs in mock.
 greenprint "📦 Installing mock requirements"
-sudo dnf -y install createrepo_c make mock python3-pip rpm-build
-
-# Install s3cmd if it is not present.
-if ! s3cmd --version > /dev/null 2>&1; then
-    greenprint "📦 Installing s3cmd"
-    sudo pip3 -q install s3cmd
-fi
+sudo dnf -y install createrepo_c make mock rpm-build s3cmd
```

Comment:
Is this change related to this PR?

Reply:
I forgot to include the explanation in the commit message while rebasing. Thanks for reminding me! I've just force-pushed an explanation, there's a copy:

s3cmd from pip doesn't work with Python 3.9. Fortunately, s3cmd in Fedora has a downstream patch to fix it.